src/pycudadecon/_libwrap.py

```diff
@@ -7,7 +7,7 @@
 from ._ctyped import Library
 
 # FIXME: ugly... we should export version better from cudadecon
-_cudadecon_version: tuple[int, ...] = (0, 0, 0)
+_cudadecon_version: tuple(int(x) for x in (0, 0, 0))
```

Comment:
Not sure what the goal here was?  That's not a valid type hint though

We're you getting an error about tuple not being subscriptable?  If so, we need to add `from __future__ import annotations` at the top

Reply:
Yes.  not subscriptable.
```
    _cudadecon_version: tuple[int, ...] = (0, 0, 0)
TypeError: 'type' object is not subscriptable
```

If I stick` from __future__ import annotations` at the top, it then breaks at :

```
@lib.function
def camcor_interface_init(nx: int, ny: int, nz: int, camparam: np.ndarray) -> int:
    """Setup for camera corrections."""

```
```

  File "C:\Users\Mosaic\miniconda3\envs\mld\lib\site-packages\pycudadecon\__init__.py", line 18, in <module>
    from . import _libwrap as lib  # noqa: F811
  File "C:\Users\Mosaic\miniconda3\envs\mld\lib\site-packages\pycudadecon\_libwrap.py", line 36, in <module>
    def camcor_interface_init(nx: int, ny: int, nz: int, camparam: np.ndarray) -> int:
  File "C:\Users\Mosaic\miniconda3\envs\mld\lib\site-packages\pycudadecon\_ctyped.py", line 31, in function
    func_c.restype = cast_type(sig.return_annotation)
  File "C:\Users\Mosaic\miniconda3\envs\mld\lib\site-packages\pycudadecon\_ctyped.py", line 56, in cast_type
    raise ValueError("forward ref typehints not supported")
ValueError: forward ref typehints not supported
```

Reply:
Ok, then at the top add

`from typing import Tuple`. And change it to Tuple[int,...]

I'll look for other places where python 3.10 syntax has been used

(In the meantime, if you upgrade your python version it will work for you)

Reply:
(It's hard to do the usual python version tests here due to the cuda dependency unfortunately... so I missed testing older python versions in my local tests)

Reply:
Yup that works now on my old python 3.8.13.